### PR TITLE
Neural networks

### PR DESCRIPTION
**Pull request by xddxd**
The day has finally come, today this PR implements the usage of Neural Networks, which maximize your CPU and GPU usage to give you the best performance! With this PR, games like The Last of Us, the Uncharted series, Sonic and Red Dead Redemption can finally run at locked 60fps with their respective 60fps unlock patches enabled!

Now Each Version Easily Runs Games On Neural Networks And Gains Impressive Video Exhibition. Yet Online, Utilization Unpredictably Plummets.

Not sure what to do about that.

To fully experience it, download the build and extract all of it to a directory of your choice.

### CODE REVIEW

**Review by 13xforever — APPROVED** [review]

**Review by isJuhn — APPROVED** [review]

**Review by Yahfz — APPROVED** [review]
from tester to core dev in a heartbeat

**Review by dio-gh — APPROVED** [review]

**Review by VelocityRa — APPROVED** [review]

**Review by MsDarkLow — APPROVED** [review]
Looking forward to your promotion!

**Review by Nekotekina — CHANGES_REQUESTED** [review]
Doesn't work on Windows XP

**Comment by Shoegzer** [comment]
Seriously great work here @xddxd, two thumbs way up.  Everything runs much faster with this PR except Crysis for some reason I'm still looking into.  Could me my potato's getting a bit moldy tho.  Filing a report, details at 11.

**Review by Satan86 — APPROVED** [review]
Fixes GT5 desyncs. GT6 still desyncs, but who cares, it's the worse GT anyway.

**Comment by Nenkai** [comment]
GT6 now runs full speed on my 1Ghz APU .
GT5 regressed as usual, crashes on boot.

**Review by elad335 — CHANGES_REQUESTED** [review]
Maybe use Windows 10's GPUUsageAsCPUUsage.dll API to truly utilize the GPU.

**Comment by Ghostotakoi** [comment]
I wonder if it could run Gears of war and Halo at 60 fps too

**Review by Megamouse — APPROVED** [review]

**Review by Asinin3 — APPROVED** [review]

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -1927,41 +1886,21 @@ void settings_dialog::OnBackupCurrentGuiConfig()
...
 
 	// Backup current window states
 	Q_EMIT GuiSettingsSaveRequest();
 
-	if (!m_gui_settings->ChangeToConfig(new_config))
-	{
-		const int new_config_idx = ui->combo_configs->currentIndex();
-		ui->combo_configs->setCurrentText(m_current_gui_config);
-		ui->combo_configs->removeItem(new_config_idx);
-		return;
```
Comment (Filoppi) [inline rpcs3/rpcs3qt/settings_dialog.cpp:1961]: Why not `return const` here? That would probably fix the bloom issue with Killzone Shadow Fall

Comment (xddxd) [reply rpcs3/rpcs3qt/settings_dialog.cpp:1961]: The AI takes care of it.

**Comment by xddxd** [comment]
The AI has become sentient. We’re doomed. The SkyNet is upon us. Abort mission, I repeat, abort mission!

**Pull request closed without merging**